modules/enterprise/server/jar/src/main/java/org/rhq/enterprise/server/configuration/ConfigurationManagerBean.java

```diff
@@ -2655,17 +2655,24 @@ private void handlePDS(final Subject subject, Resource resource, Resource parent
 
             Predicate<Resource> baseResourcePredicate = null;
             if (expressionScope == PropertyOptionsSource.ExpressionScope.BASE_RESOURCE) {
-                int halfCommaIndex = expression.indexOf(";");
-                if (halfCommaIndex > -1) {
+                String[] expressionTokens = expression.split("\\\\s+");
+                String ancestorType = null;
+                final Pattern ancestorTypePattern = Pattern.compile("(.*)\\.plugin");
```

Comment:
I'm going to use `;` to separate the value from the type,
I guess that using any of [these](https://github.com/josejulio/rhq/blob/56cab9e579247325fc815bbfbc123e40f6c95d7a/modules/enterprise/server/jar/src/main/antlr3/org/rhq/enterprise/server/search/RHQL.g#L174) will break this behavior.
e.g. `Profile\\\\;baseresource.plugin=EAP7` still yields
```sql
 ( LOWER(resource.resourceType.plugin) LIKE '%eap7%' ) 
```

@burmanm What do you think we use `value;name.plugin=EAP7` to inject extra arguments? value can be optional and left blank.